tools/triage/add_triage_signature.py

```diff
@@ -2362,6 +2362,46 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("Controller logs contain some warnings:\n" + f"{{code}}{warning_text}{{code}}")
 
 
+class CustomerHasLoggedIntoCluster(Signature):
+    """
+    This signature will detect customer login to the cluster.
+    """
+
+    USER_LOGIN_PATTERN = re.compile(r"Accepted publickey for core from")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Customer has logged into cluster nodes during installation",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+
+        cluster = md["cluster"]
+        cluster_id = cluster["id"]
+
+        triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=cluster_id)
+
+        report = ""
+        for host in cluster["hosts"]:
+                    host_id = host["id"]
+
+                    try:
+                        journal_logs = get_host_log_file(triage_logs_tar, host_id, "journal.logs")
+                    except FileNotFoundError:
+                        continue
+                    
+                    if self.USER_LOGIN_PATTERN.findall(journal_logs):
+                        report += dedent(
+                            f"""
+                        h2. Host {host_id}: found evidence of a user login during installation"""
```

Comment:
```suggestion
                        h2. Host {host_id}: found evidence of a user login during installation. <insert explanation of why this matters>"""
```

Reply:
I have updated this text to include some reason as to why this would be of interest.